lib/yapople.js

```diff
@@ -189,15 +194,17 @@ function onData(data) {
 	}
 }
 
-Client.prototype.connect = function(callback) {
+Client.prototype.connect = function(callback, options) {
```

Comment:
To use the node.js convention of the callback functions, we'll better preserve the callback as the last argument. And write something like this:
```js
Client.prototype.connect = function(options, callback) {
  if (callback === undefined) {
    callback = options;
    options = {};
  }
  ...
};
```

Reply:
@agsh : 👍 I also thought so. Since TLS options are also optional I added options as last but better be callback as final param. Fixed to reduce error scenarios. Please review.